src/lib/serviceRoutines/postUpdateContextSubscription.cpp

```diff
@@ -51,6 +51,7 @@ std::string postUpdateContextSubscription
 {
   UpdateContextSubscriptionResponse  ucsr;
   std::string                        answer;
+  bool skipInitialNotification = ciP->uriParamOptions[OPT_SKIPINITALNOTIFICATION];  
```

Comment:
I understand this correspond to `POST /v1/updateContextSubscription` operation.

Note that NGSIv1 is deprecated, no evolution on this part of the API is expected from now on. Thus, it doesn't hurt to support also the URI in this operation, but actually is not needed.

Up to you :), either response to this comment with NTC if no change is planed or revert the change and response with the corresponding `Fixed in <commithash>` comment.

Reply:
Yes @fgalan Sir, I do agree with you. But After serviceRoutine layer both NGSI v1 and v2 calls the common functionalities from mongoBackend layer. So it is necessary to make some changes in the code flow of NGSIv1. However, the changes are minimized now.

fixed in a3d00ef.